lnrpc/rpc.swagger.json

```diff
@@ -696,6 +704,15 @@
     }
   },
   "definitions": {
+    "NewAddressRequestAddressType": {
+      "type": "string",
+      "enum": [
+        "WITNESS_PUBKEY_HASH",
+        "NESTED_PUBKEY_HASH",
+        "PUBKEY_HASH"
+      ],
+      "default": "WITNESS_PUBKEY_HASH"
+    },
```

Comment:
Is this related to the rest of the pull request? It seems like it may be an unintentional reversion of an [old change](https://github.com/danrobinson/lnd/commit/129a0e10dec465d6f83c5bdd4464358172c1d6af#diff-b462ea9f4b73ab206027a14b3903d0cbL583).

Reply:
Yep, you're right about this. lnrpc/gen_protos.sh generates a bunch of code that corresponds to older commits and I missed getting rid of this one. Changing this.